en/mavgen_c/defines.md

```diff
@@ -0,0 +1,31 @@
+# MAVLink C Library #defines
+
+The following C `#defines` can be set in code in order to tune the setup for use on different platforms.
+
+- `MAVLINK_USE_CONVENIENCE_FUNCTIONS` ([protocol.h](https://github.com/ArduPilot/pymavlink/blob/master/generator/C/include_v2.0/protocol.h)): Causes convenience functions to be defined, including: `_mav_finalize_message_chan_send()`, `_mavlink_send_uart()`, `_mavlink_resend_uart()`. To use, add `#define MAVLINK_USE_CONVENIENCE_FUNCTIONS` to your code.
+- `MAVLINK_COMM_NUM_BUFFERS`: Sets the maximum number of comms buffers to be used (comms channels).
```

Comment:
Would I be correct in thinking that you might specify `1` if you knew you had a mavlink component that would only have one connection to the network?